spec/requests/portfolio_items_spec.rb

```diff
@@ -236,7 +236,7 @@
   end
 
   describe "patching portfolio items" do
-    let(:valid_attributes) { { :name => 'PatchPortfolio', :description => 'PatchDescription', :workflow_ref => 'PatchWorkflowRef', :display_name => 'Test', 'service_offering_source_ref' => "27"} }
```

Comment:
How did these display_name changes end up here?

Reply:
display_name was actually set to `readOnly` in the schema - so I removed it from the `:valid_attributes` hash. It's just ironic since they're gone from the other PR as well.

Reply:
It's just due to the common gem `Parameters` mixin, now it errors out if there are unpermitted parameters in the `params_for_update` method.